daemon/api.go

```diff
@@ -1489,7 +1489,7 @@ out:
 	// we are in charge of the tempfile life cycle until we hand it off to the change
 	changeTriggered := false
 	// if you change this prefix, look for it in the tests
-	tmpf, err := ioutil.TempFile("", "snapd-sideload-pkg-")
+	tmpf, err := ioutil.TempFile(dirs.SnapBlobDir, ".snapd-sideload-")
```

Comment:
Looks great! Do we need to ensure that we have something that cleans /var/lib/snapd/snaps/.snapd-sideload-* on bootup or periodically when e.g. power fails while such an operation is in progress?

Reply:
snapd itself could do it on startup